packages/app-desktop/gui/SearchBar/SearchBar.tsx

```diff
@@ -138,6 +138,19 @@ function SearchBar(props: Props) {
 		}
 	}, [props.notesParentType, onExitSearch]);
 
+
+	useEffect(() => {
+		if (props.notesParentType === 'Search' || props.isFocused) {
```

Comment:
Please add a comment to explain what this block does.

Reply:
It's added now